### ISSUE TITLE
Add user fiendly description for CompiledIsNullExpression and other plan nodes

### ISSUE THREAD

**Issue by eolivelli** [diennea/herddb#471]
As you can see below the CompiledIsNullExpression node does not implement "toString", and the result is the folllowing: herddb.sql.expressions.CompiledIsNullExpression@724262ba



```
herd: EXPLAIN select * from pippo where v = 1 and v is null
Executing query:EXPLAIN select * from pippo where v = 1 and v is null

| name        | value                                                                                                                                                                                                                                                                                                                                                               | 
| ----        | -----                                                                                                                                                                                                                                                                                                                                                               | 
| query       | select * from pippo where v = 1 and v is null                                                                                                                                                                                                                                                                                                                       | 
| logicalplan | LogicalProject(n=[$0], v=[$1]): rowcount = 1.0, cumulative cost = {7.0 rows, 13.0 cpu, 0.0 io}, id = 144
  LogicalFilter(condition=[AND(=($1, 1), IS NULL($1))]): rowcount = 1.0, cumulative cost = {6.0 rows, 11.0 cpu, 0.0 io}, id = 143
    EnumerableTableScan(table=[[herd, pippo]]): rowcount = 5.0, cumulative cost = {5.0 rows, 6.0 cpu, 0.0 io}, id = 142
 | 
| plan        | EnumerableInterpreter: rowcount = 5.0, cumulative cost = {2.5 rows, 2.5 cpu, 0.0 io}, id = 161
  BindableTableScan(table=[[herd, pippo]], filters=[[AND(=($1, 1), IS NULL($1))]]): rowcount = 5.0, cumulative cost = {0.025 rows, 0.03 cpu, 0.0 io}, id = 154
                                                                                                      | 
| finalplan   | BindableTableScanOp{statement=ScanStatement{table=pippo,predicate=SQLRecordPredicate{table=pippo, tableAlias=null, where=AND{[CompiledEqualsExpression{left=AccessCurrentRow{1}, right=ConstantExpression{value=1, Long}, not=false}, herddb.sql.expressions.CompiledIsNullExpression@724262ba]}, indexOp=null null},comparator=null,limits=null}}                  | 
```

**Comment by amitvc** [comment]
@eolivelli I will work on this.

**Comment by eolivelli** [comment]
Good

**Comment by amitvc** [comment]
@eolivelli With the current changes with upgrade to calcite the operators are changed a little bit

java -jar herddb-cli-0.15.0-SNAPSHOT.jar  -q "EXPLAIN select * from space.pippo where v1=100 and v1 is null";

| name        | value                                                                                                                                                                                                                                                                                                                                                               | 
| ----        | -----                                                                                                                                                                                                                                                                                                                                                               | 
| query       | select * from space.pippo where v1=100 and v1 is null                                                                                                                                                                                                                                                                                                               | 
| logicalplan | LogicalProject(pk=[$0], v1=[$1]): rowcount = 1.0, cumulative cost = {4.0 rows, 7.0 cpu, 0.0 io}, id = 155
  LogicalFilter(condition=[AND(=($1, 100), IS NULL($1))]): rowcount = 1.0, cumulative cost = {3.0 rows, 5.0 cpu, 0.0 io}, id = 153
    LogicalTableScan(table=[[space, pippo]]): rowcount = 2.0, cumulative cost = {2.0 rows, 3.0 cpu, 0.0 io}, id = 152
 | 
| plan        | EnumerableValues(tuples=[[]]): rowcount = 1.0, cumulative cost = {1.0 rows, 1.0 cpu, 0.0 io}, id = 166
                                                                                                                                                                                                                                                             | 
| finalplan   | herddb.model.planner.ValuesOp@34124f0c

**Comment by amitvc** [comment]
@eolivelli Do we want to implement toString in ValuesOp?

**Comment by eolivelli** [comment]
Maybe only reporting the number of values

**Comment by amitvc** [comment]
Ok

**Comment by eolivelli** [comment]
implemented by #600 by @amitvc
